client/state/plugins/wporg/reducer.js

```diff
@@ -1,38 +0,0 @@
-/**
- * Internal dependencies
- */
-import {
-	WPORG_PLUGIN_DATA_RECEIVE,
-	FETCH_WPORG_PLUGIN_DATA,
-	SERIALIZE,
-	DESERIALIZE
-} from 'state/action-types';
-
-function updatePluginState( state = {}, pluginSlug, attributes ) {
-	return Object.assign( {},
-		state,
-		{ [ pluginSlug ]: Object.assign( {}, state[ pluginSlug ], attributes ) }
-	);
-}
-
-function reducer( state = {}, action ) {
-	const { type, pluginSlug } = action;
-	switch ( type ) {
-		case WPORG_PLUGIN_DATA_RECEIVE:
-			if ( action.data ) {
-				return updatePluginState( state, pluginSlug, Object.assign( { isFetching: false, fetched: true, wporg: true }, action.data ) );
-			}
-			return updatePluginState( state, pluginSlug, Object.assign( { isFetching: false, fetched: false, wporg: false } ) );
-
```

Comment:
Is there ever a case where `fetched` and `wporg` do not mirror each others values?